frontend/sw360-portlet/src/main/resources/META-INF/resources/html/moderation/view.jsp

```diff
@@ -173,6 +221,151 @@
             }, [0,1,2,3,4,5,6], [7]);
         }
 
+
+        function prepareClearingRequestsData() {
+            var result = [];
+            <core_rt:forEach items="${clearingRequests}" var="request">
+                result.push({
+                    "DT_RowId": "${request.id}",
+                    "0": '',
+                    "1": '<sw360:out value="${request.projectBU}"/>',
+                    "2": "<sw360:DisplayClearingRequestLink clearingRequestId="${request.id}"/>",
+                    "3": '<sw360:out value="${request.requestedClearingDate}"/>',
+                    "4": '<sw360:DisplayUserEmail email="${request.requestingUser}" />',
+                    "5": '<sw360:DisplayUserEmail email="${request.clearingTeam}" />',
+                    "6": '<sw360:out value="${request.agreedClearingDate}"/>',
+                    "7": "<sw360:DisplayEnum value="${request.clearingState}"/>",
+                    "8": '<sw360:out value="${request.requestingUserComment}" maxChar="150" jsQuoting="true" />',
+                    "9": '<sw360:out value="${request.clearingTeamComment}" maxChar="150" jsQuoting="true" />'
+                });
+            </core_rt:forEach>
+            return result;
+        }
+
+        function prepareClosedClearingRequestsData() {
```

Comment:
Please reuse this code. As prepareClearingRequestsData() is almost same.

Reply:
I guess it's not possible, as the `result` array in function `prepareClosedClearingRequestsData()` is generated at server side using `JSTL` `forEach` tag, and I couldn't find a way to pass `item` in `forEach` loop dynamically.
Let me know if this can be achieved by any other means.